docs/src/modules/components/Demo.js

```diff
@@ -241,7 +241,9 @@ const DemoRootMaterial = styled('div', {
     borderLeftWidth: 0,
     borderRightWidth: 0,
     ...theme.applyDarkStyles({
-      backgroundColor: alpha(theme.palette.primaryDark[700], 0.1),
+      backgroundColor: theme.vars
+        ? `rgba(${theme.vars.palette.primaryDark[700]} / 0.1)`
+        : alpha(theme.palette.primaryDark[700], 0.1),
```

Comment:
@danilo-leal I did my best to only do technical modifications, but if you can double check the colors